### ISSUE TITLE
No default launch.json created

### ISSUE THREAD

**Issue by michael-hawker** [auchenberg/vscode-browser-preview#59]
If I have a new project and hit 'F5' it gives me the option to use Browser Preview as my debug target, but it will then fail as no launch.json configuration is found.

If I do the same procedure with the regular chrome debugger extension, it'll instead create the default launch.json file with the required configuration needed to launch the tools.  It'd be great for the Browser Preview to do the same.

**Comment by auchenberg** [comment]
@michael-hawker Good catch! If you press the wrench a default config should be generated for you.
